mono/metadata/mempool.c

```diff
@@ -95,6 +96,14 @@ mono_mempool_new (void)
  * \param initial_size the amount of memory to initially reserve for the memory pool.
  * \returns a new memory pool with a specific initial memory reservation.
  */
+// clang's ThreadSanitizer detects races of total_bytes_allocated and pool->d.allocated throughout the functions
```

Comment:
The comment above is some sort of doxygen / interface description - I thought it might make things clearer if I use the other kind of documentation used in that file (the `// comments ...`) for that additional information that just describes the macro.

Should I put the info about `MONO_NO_SANITIZE_THREAD` within the same comment blocks (between `* \returns ...` and `*/`) or should I open a new comment block?

Alternatively, I could also put the long text only once somewhere at the top of the file and simply explain with a few words per function to look there for further information?